src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -345,22 +359,8 @@ function NodeComponent({ handleFocusTabs, externalLinks, monitoringTools, isDevt
 
                         {params.nodeType === NodeType.Service.toLowerCase() && (
                             <div className={'col-5 pt-9 pb-9 flex left'}>
-                                {nodeName}
-                                <Tippy
-                                    className="default-tt"
-                                    arrow={false}
-                                    placement="bottom"
-                                    content={copied ? 'Copied!' : 'Copy to clipboard.'}
-                                    trigger="mouseenter click"
-                                >
-                                    <Clipboard
-                                        className="resource-action-tabs__active pl-4 icon-dim-16 pointer"
-                                        onClick={(e) => {
-                                            e.stopPropagation()
-                                            copyToClipboard(nodeName, () => setCopied(true))
-                                        }}
-                                    />
-                                </Tippy>
+                                { nodeName }
+                                {clipBoardInteraction(nodeName)}
```

Comment:
Rename it to `renderClipboardInteraction`

Reply:
updated 👍